docs/api.rst

```diff
@@ -119,6 +119,7 @@ Inference
 
 .. autosummary::
    :toctree: _generated/
+   :template: modifierclass.rst
```

Comment:
@kratsg Feelings on this? I can remove this to a separate PR, but I added it accidentally so thought I'd get your feelings in the RTD preview.

Reply:
It seems we agree this is okay for now and we can fix things up later.

Reply:
we're ok with this for now. can change it later.